eng/build.ps1

```diff
@@ -0,0 +1,35 @@
+﻿ param (
+    [string]$configuration = 'Debug'
+ )
+
+function Create-Directory([string[]] $path) {
+  if (!(Test-Path $path)) {
+    New-Item -path $path -force -itemType "Directory" | Out-Null
+  }
+}
+
+function DownloadRuntime 
+{
+    set-strictmode -version 2.0
+    $ErrorActionPreference = "Stop"
+    [Net.ServicePointManager]::SecurityProtocol = [Net.SecurityProtocolType]::Tls12  
+
+     $toolsRoot = Join-Path $PSScriptRoot "..\.tools"
+     Create-Directory $toolsRoot
+     $scriptPath = Join-Path $toolsRoot "dotnet-install.ps1"
+     Invoke-WebRequest "https://dot.net/v1/dotnet-install.ps1" -OutFile $scriptPath
+     & $toolsRoot\dotnet-install.ps1 -InstallDir $toolsRoot\dotnet -Version 2.2.203
```

Comment:
we could move version in a simple text file on eng folder and take version from there.